lib/web/ui/server.go

```diff
@@ -96,3 +97,96 @@ func MakeServers(clusterName string, servers []services.Server) []Server {
 
 	return uiServers
 }
+
+// Kube describes a kube cluster.
+type Kube struct {
+	// Name is the name of the kube cluster.
+	Name string `json:"name"`
+	// Labels is a map of static and dynamic labels associated with an kube cluster.
+	Labels []Label `json:"labels"`
+}
+
+// MakeKubes creates ui kube objects and returns a list..
+func MakeKubes(clusterName string, servers []types.Server) []Kube {
+	uiKubeClusters := []Kube{}
+	for _, server := range servers {
+		// Process each kube cluster.
+		for _, cluster := range server.GetKubernetesClusters() {
+			uiLabels := []Label{}
+
+			for name, value := range cluster.StaticLabels {
+				uiLabels = append(uiLabels, Label{
+					Name:  name,
+					Value: value,
+				})
+			}
+
+			for name, cmd := range cluster.DynamicLabels {
+				uiLabels = append(uiLabels, Label{
+					Name:  name,
+					Value: cmd.GetResult(),
+				})
+			}
+
+			sort.Sort(sortedLabels(uiLabels))
+
+			uiKubeClusters = append(uiKubeClusters, Kube{
+				Name:   cluster.Name,
+				Labels: uiLabels,
+			})
+		}
+	}
+
+	return uiKubeClusters
+}
+
+// Database describes a database server.
+type Database struct {
+	// Name is the name of the database.
+	Name string `json:"name"`
+	// Desc is the database description.
+	Desc string `json:"desc"`
+	// Procotol is the database description.
+	Procotol string `json:"protocol"`
+	// Type is the database type, self-hosted or AWS RDS.
```

Comment:
Nit: Instead of "or AWS RDS" I would just say "or cloud-hosted" which would cover RDS, Redshift, Cloud SQL and anything we'll add in future.